Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -136,8 +136,14 @@ def set_data(self, data):
         self.set_input_summary()
 
         if data is not None and data.domain.class_var is None:
-            self.Error.data_error("Data has no target variable.\n"
-                                  "You can set a target variable with the Select Columns widget.")
+            if data.domain.class_vars:
+                self.Error.data_error(
+                    "Data contains multiple target variables.\n"
+                    "You can select a single one with the Select Columns widget.")
```

Comment:
"You can select a single one with the Select Columns widget." --> "Select a single one with Select Columns." (shorter?)

Reply:
"You can" was meant to sound as suggestion, just one of possible options. It obviously didn't, so I removed it.

Reply:
I started from the position that I would like to use the learner, so I *have* to have just a single target variable. 😁